### ISSUE TITLE
Snippet Highlighting also selects line numbers.

### ISSUE THREAD

**Issue by bwiggs** [kkuchta/Vimbits#6]
I find it difficult to copy code snippets from the site. When I try to highlight to select i also end up highlight the line number.

**Comment by bwiggs** [comment]
NEVERMIND, it just looks like it's highlighting the line numbers, but it doesn't save them to the clipboard

**Comment by kkuchta** [comment]
Yep.  Planning a better solution- maybe a simple "Put in clipboard" button, but this works for now.
